tests/build.gradle

```diff
@@ -98,6 +100,16 @@ ext.testSets = [
         "includes" : [
             "system/basic/**"
         ]
+    ],
+    "REQUIRE_LEAN_SYSTEM" : [
+        "includes" : systemIncludes,
+        
+        // Lean OpenWhisk doesn't use Kafka and has no Kafka installed as part of the setup.
+        // Tests suit below validating KafkaConnector so have to be excluded for Lean System tests
```

Comment:
`         // Tests suits below require Kafka so they are excluded for Lean System tests`